src/models/user-model.ts

```diff
@@ -6,6 +6,7 @@ import Attributes from './model';
  * User attributes interface.
  */
 export interface UserAttributes extends Attributes {
+    [key: string]: string | Date | number | string[] | unknown;
```

Comment:
Another solution instead of that ?

Reply:
No I'm really sorry but that it is not going to be possible Mr. Surieux.
You will have to stick with that attribute